script/single-deployment/DeployAAVEBorrower.s.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.28;
+
+import {Script, console} from "forge-std/Script.sol";
+
+import {AaveBorrower} from "../../src/AaveBorrower.sol";
+
+import {FlashLoanRouter} from "./DeployFlashLoanRouter.s.sol";
+
+import {Constants} from "script/libraries/Constants.sol";
+
+/// @title Deploy Aave Borrower
+/// @author CoW DAO developers
+/// @notice A contract that deploys a borrower contract that adds support for
+/// Aave protocol.
+contract DeployAAVEBorrower is Script {
+    /**
+    * @dev Executes the deployment of the `AaveBorrower` contract.
+    * 
+    * This function calls the `deployAAVEBorrower` method with `FlashLoanRouter(address(0))`,
+    * which triggers the retrieval of the `FlashLoanRouter` address from the environment variable 
+    * `FLASHLOAN_ROUTER_ADDRESS`.
+    */
+    function run() public virtual {
+        deployAAVEBorrower(FlashLoanRouter(address(0)));
```

Comment:
nanonit: move reading form env var up here to avoid loading `deployAAVEBorrower` with the extra conditional of treating address(0)